src/overlay/OverlayMetrics.cpp

```diff
@@ -83,6 +83,10 @@ OverlayMetrics::OverlayMetrics(Application& app)
           app.getMetrics().NewTimer({"overlay", "delay", "write-queue"}))
     , mMessageDelayInAsyncWriteTimer(
           app.getMetrics().NewTimer({"overlay", "delay", "async-write"}))
+    , mMessageDelayInWriteQueueTimerSlow(
+          app.getMetrics().NewTimer({"overlay", "delay", "write-queue-slow"}))
+    , mMessageDelayInAsyncWriteTimerSlow(
```

Comment:
I think that we'll evolve over time how we want to categorize peers. I think that "quality" is probably the dimension we're going after (deriving for now from "latency") so the suffix should be `low` (for low quality).

Should there be, `foo` (for all), `foo-high` (for high) and `foo-low` (for low) ? I am a bit worried that if anything is wrong with quality we end up with unusable metrics as we can't reconstruct "all".